src/cloud/project/project-patch.md

```diff
@@ -111,3 +111,7 @@ To use magento-cloud-patches as a stand-alone package:
    You can clean the cache using the Magento Admin Cache Management.
 
 1. Test the patches, and make any necessary changes to custom patches.
+
+## Skip Cloud patches
+
+You cannot skip required patches on Cloud projects.
```

Comment:
Hi @atwixfirster. This info is definitely useful and might help developers to avoid confusions. But it would be even more helpful if we could extend the info a little bit by explaining what are "required patches" in terms of Magento Commerce Cloud. 

What do you think?

Reply:
@rogyar , I 've added a link to https://github.com/magento/magento-cloud-patches/tree/develop/patches. I think it explains better than any words :)

Thank you for notice!

Reply:
@atwixfirster I don't think this requires a separate section.  How about changing to a note and adding after line 16?

{:.bs-callout-info}
When you update {{ site.data.var.ct }} or the {{ site.data.var.mcp-prod }} package, the latest required patches are applied automatically the next time you deploy your project. You cannot skip required patches, or remove them after they have been applied.

Also, we'll be updating this topic for the next release to improve the instructions and clarify the different types of patches (required, hotfixes, and custom).

Reply:
Hi, @meker12 !

Your suggestions have been implemented. However, I have edited your suggestions about `You cannot skip required patches, or remove them after they have been applied.`.

We cannot skip a required patches during a deployment process. 

We have the ability to revert changes which patches have made via patches from `m2-hotfixes` folder. Cloud executes patches from `m2-hotfixes` folder when a required patches have been applied.

Please review a proposed changes.

Thank you!